### PR TITLE
post: ng perf series part 2

### PR DESCRIPTION
**Pull request by cexbrayat**

### CODE REVIEW

File: _posts/2018-09-13-angular-performances-part-2.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+So, once your user has opened the application once,
+it's possible to speed the following reloading.
+Let's see how.
+
+## Caching
+
+You should always cache the assets of your application (images, styles, JS bundles...).
+This is done by configuring your server and leveraging the `Cache-Control` and `ETag` headers.
+All the servers of the market allows to do so,
```
Comment (clacote) [inline _posts/2018-09-13-angular-performances-part-2.md]: allows -> allow

File: _posts/2018-09-13-angular-performances-part-2.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Angular Performances Part 2 - Reload
+author: cexbrayat
+tags: ["Angular 6", "Angular 5", "Angular", "Angular 2", "Angular 4", "Angular CLI", "performances", "benchmarks"]
+description: "Learn how to make your Angular application faster. In this second part, let's talk about whow you can speed up the reloading of an application."
```
Comment (jnizet) [inline _posts/2018-09-13-angular-performances-part-2.md]: whow -> how

File: _posts/2018-09-13-angular-performances-part-2.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+---
+
+This is the second part of this series (check the [first post](/2018/09/06/angular-performances-part-1) if you missed it), and this blog post is about how you can speed up the reloading of an Angular application.
+In future posts, we'll talk about how to profile your running application,
+and how to improve runtime performances.
+If you are the lucky owner of our ebook, you can already check the other parts if you download the [last ebook release](https://books.ninja-squad.com/claim?book=Angular).
+
+
+So, once your user has opened the application once,
+it's possible to speed the following reloading.
```
Comment (jnizet) [inline _posts/2018-09-13-angular-performances-part-2.md]: Duplication of once.
I'd rephrase as a question to:
So, let's assume a user uses your application for the first time. How to make sure that, when he comes back later, the application starts even faster?

Comment (cexbrayat) [reply _posts/2018-09-13-angular-performances-part-2.md]: 👍 I went with
